### DISCUSSION TITLE
Can x64 / x86 games be installed in a sanbox and be fully emulated on ARM Windows 11?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [sandboxie-plus/Sandboxie#3113] [General]
I've been trying to get this old game [Blackhawk Striker 2 (2004)](https://www.wildtangent.com/p/arcade/blackhawk-striker-2/) running on Windows 11 on ARM in Parallels but have had no luck, even when I set all the .exes for WildTangent and the game itself to "very strict emulation" in the Windows properties dialogs.

**Comment by bastik-1001** [comment]
Unless the ARM version of Sandboxie, does something I am not aware of, it will not help you to make something run that does not run on the un-sandboxed system or within software that emulates another environment.

When there is something that runs these games, it might be possible to run that under the supervision of Sandboxie. There could be compatibility issues and a performance impact.
